README.md

```diff
@@ -109,6 +111,13 @@ Copy the following files to `files/lib` directory.
 1. [MySQL Connector/J](https://dev.mysql.com/downloads/connector/j/5.1.html)
 2. [Amazon Coretto for Linux x64 JDK](https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/downloads-list.html)
```

Comment:
It should be Corretto

Reply:
Fixed https://github.com/wso2/ansible-apim/pull/43/commits/ae9d3525e0a4c7d549bc3aeb08854603abcb5101